saritasa_drf_tools/filters/ordering.py

```diff
@@ -146,3 +146,40 @@ def _validate_ordering_fields(
                 f" or non-related fields for action {view_action}."
                 f" {error}",
             )
+
+
+class OrderingFilterBackendWithSecondarySorting(OrderingFilterBackend):
```

Comment:
I think name is too long. Try to find shorter one, like `OrderingWithPKFilterBackend` or `DetermenisticOrderingFilterBackend`